static/js/templates.js

```diff
@@ -2,6 +2,14 @@ var templates = (function () {
 
 var exports = {};
 
+function trans(str, context) {
+    if (typeof i18n !== 'undefined') {
+        // only translate the string if i18n is loaded
+        return i18n.t(str, context);
+    }
+    return str;
+}
```

Comment:
I'm a little worried about this change -- doesn't it risk problems in production where things that load too early don't show up as translated in a nondeterministic way?  Maybe I'm missing something about how this is used...